src/components/ComponentPages/ProfileInfo/ProfileInfoUI/index.tsx

```diff
@@ -0,0 +1,268 @@
+import React from "react";
+import { Row, Col, Typography, Form, Input, Button, Select, DatePicker, Radio } from "antd";
+import styles from "./ProfileInfo.module.scss";
+import messages from "../../../../messages";
+
+const { Title, Text } = Typography;
+const { Option } = Select;
+
+function ProfileInfoUI() {
+
+  const config = {
+    rules: [
+      {
+        type: 'object',
+        required: true,
+        message: 'Please select time!',
+      },
+    ],
+  };
+  const selectAfter = (
+    <Select defaultValue="private" className="select-after">
+      <Option value="private">Private</Option>
+      <Option value="public">Public</Option>
+    </Select>
+  );
+
+
+  return (
+    <section className={styles.profileInfo_wrapper}>
+      <Form
+        name="verifyNumber"
+        initialValues={{ prefix: "+91" }}
+        layout="vertical"
+      >
+        <div className={styles.section_one}>
+          <Row gutter={30}>
+            <Col md={12}>
+              <Form.Item
+                name="phoneNumber"
+                label={messages.labels.phoneNumber}
+                rules={[
+                  {
+                    required: true,
```

Comment:
Try to move validation into its own component or package

Reply:
Done